stdlib/SuiteSparse_LBT_jll/Project.toml

```diff
@@ -0,0 +1,14 @@
+name = "SuiteSparse_LBT_jll"
```

Comment:
What is this?

Reply:
This is the SuiteSparse that links to libblastrampoline, instead of openblas. We could either update SuiteSparse_jll to start linking to openblas, or have a separate JLL. For now, I just made it a separate JLL to keep it simple. Is there a preference on the right way to do this?

Reply:
I think what I need to do is update the SuiteSparse.jl Project.toml to include a dependency on SuiteSparse_jll / SuiteSparse_LBT_jll. I don't see that it does that right now, but it probably should.

Reply:
@staticfloat Is replacing SuiteSparse_jll with SuiteSparse_LBT_jll a problem? Will it affect downstream packages that depend on SuiteSparse_jll?

Or should we work on a new SuiteSparse_jll (which anyways needs a bump in version numbers) and we start linking to LBT in a new version which is for Julia 1.7+

Reply:
I think making a `SuiteSparse_jll` that links against LBT is probably the best way forward; make it as Julia v1.7+ only, and we can experiment with it on `master` for a bit.